modules/prebidServerBidAdapter/index.js

```diff
@@ -626,10 +626,17 @@ const OPEN_RTB_PROTOCOL = {
         if (adapter && adapter.getSpec().transformBidParams) {
           bid.params = adapter.getSpec().transformBidParams(bid.params, isOpenRtb());
         }
+
         acc[bid.bidder] = (_s2sConfig.adapterOptions && _s2sConfig.adapterOptions[bid.bidder]) ? Object.assign({}, bid.params, _s2sConfig.adapterOptions[bid.bidder]) : bid.params;
         return acc;
       }, {});
 
+      // if it exists and is a video request, pass the storedAuctionResponse
+      const storedAuctionResponse = utils.deepAccess(bidRequests, '0.bids.0.storedAuctionResponse');
+      if (storedAuctionResponse && mediaTypes.video) {
+        utils.deepSetValue(ext, 'prebid.storedauctionresponse', storedAuctionResponse);
+      }
```

Comment:
1.  why only video?  any PBS request should be supported
2.  I may not understand the data structure, but why are you only looking in the first `bid` of the first `bidRequest` (`0.bids.0.storedAuctionResponse`)?